components/02-molecules/edit/FieldsContext.tsx

```diff
@@ -211,7 +262,56 @@ const FieldsProvider: React.FC<FieldsProviderProps> = ({ children }) => {
 
         return fieldIsEmpty || isAddressValid;
       },
-    } as Field,
+    },
+    {
+      label: "btc",
+      placeholder: "bc1000000000000000000000000000000000000000",
+      fieldType: FieldType.Address,
+      value: "",
+      validationFunction: (fieldValue: string) => {
+        const fieldIsEmpty: Readonly<boolean> = fieldValue === "";
+        const isValidAddress: Readonly<boolean> =
+          typeof fieldValue === "string" &&
+          !!validateBitcoinAddress(fieldValue);
+        return fieldIsEmpty || isValidAddress;
+      },
+    },
+    {
+      label: "arb1",
+      placeholder: "0x0000000000000000000000000000000000000000",
+      fieldType: FieldType.Address,
+      value: "",
+      validationFunction: (fieldValue: string) => {
+        const fieldIsEmpty: Readonly<boolean> = fieldValue === "";
+        const isValidAddress: Readonly<boolean> =
+          typeof fieldValue === "string" && !!isAddress(fieldValue);
+        return fieldIsEmpty || isValidAddress;
+      },
+    },
+    {
+      label: "opt",
+      placeholder: "0x0000000000000000000000000000000000000000",
+      fieldType: FieldType.Address,
+      value: "",
+      validationFunction: (fieldValue: string) => {
+        const fieldIsEmpty: Readonly<boolean> = fieldValue === "";
+        const isValidAddress: Readonly<boolean> =
+          typeof fieldValue === "string" && !!isAddress(fieldValue);
+        return fieldIsEmpty || isValidAddress;
+      },
+    },
+    {
+      label: "matic",
+      placeholder: "0x0000000000000000000000000000000000000000",
+      fieldType: FieldType.Address,
+      value: "",
+      validationFunction: (fieldValue: string) => {
+        const fieldIsEmpty: Readonly<boolean> = fieldValue === "";
+        const isValidAddress: Readonly<boolean> =
+          typeof fieldValue === "string" && !!isAddress(fieldValue);
```

Comment:
Why are we checking if fieldValue === "string" here? That should be defined as a string already
`validationFunction: (fieldValue: string) => {`

Reply:
Good point, I'll remove this verification from all the fields